src/View/Home/see_artworks.html.twig

```diff
@@ -0,0 +1,33 @@
+
+<div class="card-deck">
+
+    {% for seeArtwork in seeArtworks %}
```

Comment:
artwork in artworks

Reply:
j'ai supprimé les fichiers indésirables ds ce PR..